main/py-freezegun/APKBUILD

```diff
@@ -0,0 +1,65 @@
+# Maintainer: Drew DeVault <sir@cmpwn.com>
+pkgname=py-freezegun
+_pyname=freezegun
+pkgver=0.3.11
+pkgrel=0
+pkgdesc="Let your Python tests travel through time"
+url="https://github.com/spulec/freezegun"
+arch="noarch"
+license="Apache-2.0"
+_py2_deps="py2-six py2-dateutil"
+_py3_deps="py3-six py3-dateutil"
+makedepends="
+	$_py2_deps $_py3_deps python2-dev python3-dev py2-setuptools py3-setuptools
```

Comment:
package is noarch i think the -dev packages can be removed.

Reply:
Thanks (again) :smile: I did not look at this as I just `git am`-ed it from patchwork.

Removed -dev and fixed the commit msg.